### PR TITLE
Add ARM64 blog post

### PR DESCRIPTION
**Pull request by kkraune**
.. or @frodelu

The post is approved

### CODE REVIEW

File: _posts/2022-10-11-vespa-on-arm64.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+width="500" height="auto" alt="Decorative image" />
+
+<p class="image-credit">
+Photo by <a href="https://unsplash.com/es/@niekdoup?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Niek Doup</a>
+on <a href="https://unsplash.com/s/photos/cpu-chip?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a>
+</p>
+
+Since open-sourcing [Vespa.ai](https://vespa.ai/) in 2017,
+it has been available as a container image for the x86_64 architecture.
+Vespa.ai is now released as a multiplatform image, supporting both x86_64 and ARM64.
```
Comment (frodelu) [inline _posts/2022-10-11-vespa-on-arm64.md:22]: Did we explicitly choose to go for ARM64 vs. aarch64? I believe the latter is the official name for the architecture while many colloquially use ARM64.

Comment (kkraune) [reply _posts/2022-10-11-vespa-on-arm64.md:22]: yes, please see Arnstein's comments in the ARM work package (open the closed comments).

**Comment by kkraune** [comment]
@aressem pls approve

**Review by aressem — APPROVED** [review]

**Comment by aressem** [comment]
Link checker complains about https://docs.vespa.ai/en/build-install-vespa.html#docker-images

**Pull request merged**